### DISCUSSION TITLE
Android- how to display a bitmap on screen?

### DISCUSSION THREAD

**Discussion by ShakedDovrat** [korlibs/korge#483] [Q&A] [answered]
Hi,
I have a Bitmap32 object which contains data of some image. I have a View object in Android UI.
I'm trying to display the image on that View. Korim documentation didn't help.
Please advise

**Comment by soywiz** [comment, marked as answer]
You can try: https://github.com/korlibs/korge/blob/b068eb7c57dcd7794247e7f393d2a15feb6356ea/korim/src/androidMain/kotlin/com/soywiz/korim/format/NativeImageFormatProviderAndroid.kt#L129

to convert a Bitmap from korim into a `android.graphics.Bitmap`, then use the Android standard functionality to display that Bitmap

**Comment by ShakedDovrat** [comment]
Thank you for your answer. What is the equivalent for converting a Bitmap32 to an iOS image? `createBitmapNativeImage`?

**Reply by soywiz** [reply]
There is no equivalent right now IIRC, but there is this function:

https://github.com/korlibs/korge/blob/1fe2c8a34949b57f5971346b0443549dc6e310af/korim/src/darwinMain/kotlin/com/soywiz/korim/format/cg/CgRenderer.kt#L20

That you can use to transfer a `Bitmap32` into a `CGContextRef`
